icevision/models/ross/efficientdet/dataloaders.py

```diff
@@ -92,26 +92,22 @@ def build_train_batch(records, batch_tfms=None):
     ```
     """
     records = common_build_batch(records, batch_tfms=batch_tfms)
+    batch_images, batch_bboxes, batch_classes = zip(
+        *(process_train_record(record) for record in records)
+    )
 
-    images = []
-    targets = {"bbox": [], "cls": []}
-    for record in records:
-        image = im2tensor(record["img"])
-        images.append(image)
-
-        if len(record["labels"]) == 0:
-            targets["cls"].append(tensor([0], dtype=torch.float))
-            targets["bbox"].append(tensor([[0, 0, 0, 0]], dtype=torch.float))
-        else:
-            labels = tensor(record["labels"], dtype=torch.float)
-            targets["cls"].append(labels)
-
-            bboxes = tensor([bbox.yxyx for bbox in record["bboxes"]], dtype=torch.float)
-            targets["bbox"].append(bboxes)
+    # convert to tensors
+    batch_images = torch.stack(batch_images)
+    batch_bboxes = [Tensor(bboxes) for bboxes in batch_bboxes]
+    batch_classes = [Tensor(classes) for classes in batch_classes]
 
-    images = torch.stack(images)
+    # convert to EffDet interface
+    targets = dict(
+        bbox=batch_bboxes,
+        cls=batch_classes,
```

Comment:
```suggestion
        cls=batch_classes
```

This way it stays in one line =)